### DISCUSSION TITLE
Controlling the keypad using json

### DISCUSSION THREAD

**Discussion by DaveGalbraith** [technyon/nuki_hub#536] [Q&A]
I'm still having issues controlling the keypad using json (using version 9.02-beta9). I can add and delete entries (so I know I have the correct topic and json structure) but I can't update (usually get commandResultJson returning failed, or e.g invalidAllowedFromTime in the event I don't set that during update action).

Here is my MQTT Publish set-up:

![image](https://github.com/user-attachments/assets/cc443ed6-3697-4f7c-8910-ab7cbf8918af)

Is there a way to debug this ( I have MQTT logs on but they don't tell me an awful lot)

**Comment by iranl** [comment]
Closing as this is now an issue #563
